modules/promo-code/promo-code.resolver.js

```diff
@@ -4,6 +4,9 @@ const promoCodeQuery = {
   getAllPromoCodes: () => PromoCodeService.getAllPromoCodes(),
 
   getPromoCodeById: async ({ id }) => PromoCodeService.getPromoCodeById(id),
+
+  getPromoCodeByCode: async (_, { code }) =>
+    PromoCodeService.getPromoCodeByCode(code),
```

Comment:
why `PromoCodeService`? it isn't class name, just an instance. just `promoCodeService`

Reply:
will be done

Reply:
Fixed